syntax = "proto3";

package AddService;

service AddService {
  rpc Add (AddRequest) returns (AddResponse) {}
  rpc PlusPlus (PlusPlusRequest) returns (AddResponse) {}
  rpc Ping (PingRequest) returns (PingResponse) {}
}

message AddRequest {
  int32 num1 = 1;
  int32 num2 = 2;
}
message PlusPlusRequest {
  int32 num = 1;
}

message AddResponse {
  int32 num = 1;
}
message PingRequest {}
message PingResponse {}
